md-docs/user_guide/project.md

```diff
@@ -0,0 +1,53 @@
+# Project
+
+A Project is the secondary organizational entity in ML cube Platform.
+A Project groups together a set of artificial intelligence algorithms that share a common goal measured by a set of KPIs.
+For this reason, it is composed of several [Tasks] that collaborate to reach the Project's goal.
+
+Users in the [Company] can access to one or more Projects according to their [roles].
+
+## Creation
+
+When a Project is created, the [User] specifies its *name*, and *description*, and selects the *default storage policy*.
```

Comment:
Troppi `and` forse?

Reply:
me l'ha detto Grammarly ma anche a me non piaceva haahah